external/emulation/emulation/_emulate/microphysics.py

```diff
@@ -104,7 +130,7 @@ def microphysics(self, state: FortranState) -> None:
 
         # grab model-required variables and
         # switch state to model-expected [sample, feature]
-        inputs = [state[name].T for name in self.model.input_names]
+        inputs = {name: state[name].T for name in self.model.input_names}
```

Comment:
Is this just to make the I/O more congruent?

Reply:
I don't think was necessary, but I hopefully it makes it a bit more robust to order or argument errors.